src/server/webkit/wkPage.ts

```diff
@@ -591,6 +591,14 @@ export class WKPage implements PageDelegate {
       }
       promises.push(session.send('Page.setForcedAppearance', { appearance }));
     }
+    if (reducedMotion !== null) {
```

Comment:
Interestingly, we never reset to "system default" in this method for both reduced motion and color scheme.

Reply:
Good catch, probably hard to add there a system-test since GitHub Action environments are all quite different.